actionpack/lib/action_controller/api.rb

```diff
@@ -0,0 +1,157 @@
+require 'action_view'
+require 'action_controller'
+require 'action_controller/log_subscriber'
+
+module ActionController
+  # API Controller is a lightweight version of <tt>ActionController::Base</tt>,
+  # created for applications that don't require all functionality that a complete
+  # \Rails controller provides, allowing you to create faster controllers for
```

Comment:
"faster controllers" I don't believe it is true and even it is we know that it would matter in real world. I'd change it to `allowing you to create controllers with just the features that you need for API only applications`.